### DISCUSSION TITLE
Observed changes in recent versions of fheroes2 in mission 8 Price of Loyalty and the current difficulty of completing this mission.

### DISCUSSION THREAD

**Discussion by TLCD17** [ihhub/fheroes2#9110] [General]
At the very beginning, I wanted to thank all the people who programmed the fheroes2 game and who contributed to its improvement.

I want to say that I have a lot of experience playing various HOMM series, but I also do not know everything.

Now I will go straight to the main reason why I am writing this topic.

In the latest version of fheroes2 - 1.1.2 (September 15, 2024) in the Price of Loyalty campaign - mission 8 I had a better start than ever before. Unfortunately, minotaurs inflicted large initial losses on me every time I started this mission in the latest version. Despite having the best army I managed to get, I could not defeat a single starting castle in the nearest areas on any closest island from the starting island.

Already a few versions back of fheroes2, as well as the latest versions were easier to exploit in mission 8, although it was still difficult, but I could not win this mission.
While searching the internet how to beat this mission, I came across this video:
https://www.youtube.com/watch?v=wAVDhRQExFs&t=728s

If the person on YouTube in this video has the Heroes 2 HD version, I would like to have that version of the game. I am guessing that it is one of the fheroes2 versions that was released 2 years ago, specifically: version 0.9.18 (August 10, 2022).

With this version of the game - if I could get it, I could play this version and see if this mission is really so hard to beat that it is impossible to win, or if there is some way to beat this mission. Right now it is one of the hardest, especially after so many changes and modifications made to this mission that it is not what it should be in reality.

**Comment by ihhub** [comment]
This is not a question or issue. Moving it to discussion section.

**Comment by oleg-derevenetz** [comment]
> Unfortunately, minotaurs inflicted large initial losses on me every time I started this mission in the latest version.

To be honest, Minotaurs (at the starting island?) can only become a problem here if there is an upgraded stack, because it's too fast. Otherwise there is no problem at all if you have the Elven Alliance award (after the 3rd mission) - just join the group of Elves near the gold mine northwest of the castle and voila:

https://github.com/user-attachments/assets/30f1fc12-4207-4397-bf19-43bb5626fd0f

If the battle with the Minotaurs can still be difficult due to the upgraded stack, then just attack the Hydras, even the instant battle will win without any losses at all:

https://github.com/user-attachments/assets/87c14527-5722-47c7-b657-9e2cc7087993

**Reply by fheroes2bugs** [reply]
> just join the group of Elves

What is the condition of joining them? I tried like 50 times restarting the mission, but they don't join me.

**Reply by oleg-derevenetz** [reply]
> > just join the group of Elves
> 
> What is the condition of joining them? I tried like 50 times restarting the mission, but they don't join me. 

> there is no problem at all if you have the Elven Alliance award (after the 3rd mission)

**Comment by TLCD17** [comment]
@oleg-derevenetz Thanks for the tips! but is there a eventuality to download older versions of fheroes2?

**Reply by zenseii** [reply]
Hi, @TLCD17. All previous releases can be found in the "Releases" area of this repo https://github.com/ihhub/fheroes2/releases

As for an older version being closer to what this mission in your words "should be", which I believe means play like it did in the original game, then I can already say that this will not be the case, neither with version 0.9.18 nor the latest current fheroes2. This is because a lot of factors determine how a scenario plays out, and there are many differences between the original AI and the fheroes2 AI. These are things like bonuses/cheats that the original AI had, AI decision-making etc.

If you want to experience the map's difficulty as it was in the original game then you have to play it with the original game. We can't bring the original AI into fheroes2.

fheroes2 had to make a new AI from scratch because none of the code is based on the original game's code. The promise of fheroes2 is to bring a new experience by providing a new AI.

Afaik, you did not say which difficulty you were playing on, but one advice would be to lower it if possible.

Here's a series of relatively short videos of someone beating this on version 1.1.1 :
https://youtu.be/83mwVae2y6g?si=bIbZx93iMNQVXEZJ

**Reply by fheroes2bugs** [reply]
> https://youtu.be/83mwVae2y6g?si=bIbZx93iMNQVXEZJ

Are you sure this is 1.1.1?

**Reply by zenseii** [reply]
@TLCD17, yes I asked the maker of the video.

**Comment by TLCD17** [comment]
Thank you very much to everyone for your replies regarding my topic. I am closing the topic because the problem has been solved.
